manticore/ethereum.py

```diff
@@ -1224,13 +1225,15 @@ def flagged(flag):
                         is_argument_symbolic = any(map(issymbolic, arguments))
                         is_something_symbolic = is_something_symbolic or is_argument_symbolic
                         tx_summary.write(') -> %s %s\n' % ( tx.result, flagged(is_argument_symbolic)))
-                        is_return_symbolic = any(map(issymbolic, return_data))
-                        return_values = tuple(map(state.solve_one, return_data))
-                        if len(return_values) == 1:
-                            return_values = return_values[0]
 
-                        tx_summary.write('return: %r %s\n' % ( return_values, flagged(is_return_symbolic)))
-                        is_something_symbolic = is_something_symbolic or is_return_symbolic
+                        if return_data is not None:
```

Comment:
can we restructure this so that there is only one if block? move the `if tx.result == 'RETURN'` below the block above (the block that emits the "Function call:" text etc)?